src/components/data_submission/ds_study_information.js

```diff
@@ -0,0 +1,144 @@
+import { useState, useEffect } from 'react';
+import { h, h2, div } from 'react-hyperscript-helpers';
+import { Notifications, isEmailAddress } from '../../libs/utils';
+import { User } from '../../libs/ajax';
+import { FormFieldTypes, FormField, FormTable, FormValidators } from '../forms/forms';
+
+export default function DataSubmissionStudyInformation(props) {
+  const { onChange } = props;
+  const [user, setUser] = useState();
+
+  //init hook, need to make ajax calls here
+  useEffect(() => {
+    const updateUserAndFields = async () => {
+      const me = await User.getMe();
+      setUser(me);
+      onChange({key: 'dataSubmitterId', value: me.dacUserId});
+    };
+
+    const init = async () => {
+      try {
+        updateUserAndFields();
+      } catch (error) {
+        Notifications.showError({text: 'Error: Unable to retrieve user data from server'});
+      }
+    };
+
+    init();
+  }, [onChange]);
+
+  return h(div, {
+    style: {
+      padding: '50px 0',
+      maxWidth: 800,
+      margin: 'auto'
+    }
+  }, [
+    h2('Study Information'),
+    h(FormField, {
+      id: 'studyName',
+      title: 'Study Name',
+      validators: [FormValidators.REQUIRED],
+      onChange
+    }),
+    h(FormField, {
+      id: 'studyType',
+      title: 'Study Type',
+      type: FormFieldTypes.SELECT,
+      selectOptions: [
+        'Observational', 'Interventional', 'Descriptive',
+        'Analytical', 'Prospective', 'Retrospective',
+        'Case report', 'Case series', 'Cross-sectional',
+        'Cohort study'
+      ],
+      onChange
+    }),
+    h(FormField, {
+      id: 'studyDescription',
+      title: 'Study Description',
+      placeholder: 'Description',
+      onChange
+    }),
+    h(FormField, {
+      id: 'dataTypes',
+      title: 'Data Types',
+      placeholder: 'Type',
+      defaultValue: [],
+      type: FormFieldTypes.MULTITEXT,
+      onChange
+    }),
+    h(FormTable, {
+      id: 'fileTypes',
+      formFields: [
+        {
+          id: 'fileType',
+          title: 'File Type',
+          type: FormFieldTypes.SELECT,
+          selectOptions: ['Arrays', 'Genome', 'Exome', 'Survey', 'Phenotype']
+        }, {
+          id: 'functionalEquivalence',
+          title: 'Functional Equivalence',
+          placeholder: 'Type'
+        }, {
+          id: 'numberOfParticipants',
+          title: '# of Participants',
+          placeholder: 'Number',
+          type: FormFieldTypes.NUMBER
+        }
+      ],
+      defaultValue: [{}],
+      onChange
+    }),
+    h(FormField, {
+      id: 'phenotypeIndication',
+      title: 'Phenotype/Indication Studied',
+      onChange
+    }),
+    h(FormField, {
+      id: 'species',
+      title: 'Species',
+      onChange
+    }),
+    h(FormField, {
+      id: 'piName',
+      title: 'Principal Investigator Name',
+      onChange
+    }),
+    user && h(FormField, {
```

Comment:
Nit: The props object for hyperscript functions can process an `isRendered` key for conditional rendering. so you can do something like...

```
h(FormField, {
      id: 'dataSubmitterEmail',
      title: 'Data Submitter Email',
      defaultValue: user?.email,
      disabled: true,
      isRendered: !isEmpty(user),
      onChange
    })
```

and have the same effect.